L/LibGit2/build_tarballs.jl

```diff
@@ -59,10 +58,10 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("MbedTLS_jll", v"2.24.0"),
-    Dependency("LibSSH2_jll"),
+    Dependency("MbedTLS_jll"; compat="~2.28.0"),
+    Dependency("LibSSH2_jll"; compat="~1.10.1"),
```

Comment:
I think this is too restrictive: contrary to `MbedTLS`, I'm not aware of the fact libssh2 routinely breaks API/ABI in every minor version.  Good to have a compat, but I'd relax it to
```suggestion
    Dependency("LibSSH2_jll"; compat="1.10.1"),
```